includes/wikia/helpers/FileNamespaceSanitizeHelper.php

```diff
@@ -88,12 +88,8 @@ private function convertToPlainText( $filename ) {
 	private function extractFilename( $potentialFilename, $filePrefixRegex ) {
 		$trimmedFilename = trim( $potentialFilename, "[]" );
 		$unprefixedFilename = mb_ereg_replace( $filePrefixRegex, "", $trimmedFilename );
-		$filenameParts = explode( '|', $unprefixedFilename );
-		if ( !empty( $filenameParts[ 0 ] ) ) {
-			return urldecode( $filenameParts[0] );
-		}
 
-		return null;
+		return self::removeImageParams( $unprefixedFilename );
```

Comment:
Now I'm wondering if extractFilename could be rewritten to use getCleanFileMarkersFromWikitext, thus removing the need to have removeImageParams completely